api/src/main/java/io/strimzi/api/kafka/model/tracing/Tracing.java

```diff
@@ -34,8 +34,7 @@ public abstract class Tracing implements UnknownPropertyPreserving, Serializable
     private Map<String, Object> additionalProperties;
 
     @Description("Type of the tracing used. " +
-            "Currently the only supported types are `jaeger` for OpenTracing (Jaeger) tracing and `opentelemetry` for OpenTelemetry tracing. " +
-            "The OpenTracing (Jaeger) tracing is deprecated")
+            "Currently the only supported type is `opentelemetry` for OpenTelemetry tracing.")
```

Comment:
I think maybe we could add something like we have for example here? https://github.com/strimzi/strimzi-kafka-operator/blob/a2bae6f2498f187253d189d61c8082a3ea6b8c8b/api/src/main/java/io/strimzi/api/kafka/model/KafkaSpec.java#L102

That `jaeger` will be ignored from 0.37.0 on?